charts/hedera-json-rpc-relay/values.yaml

```diff
@@ -62,7 +62,7 @@ cronjob:
     pullPolicy: IfNotPresent
     repository: postman/newman
     tag: 5.3.1-alpine
-  postman: '' # Custom postman.json in base64 encoding
+  postman: ""  # Custom postman.json in base64 encoding
```

Comment:
what does this fix?

Reply:
old commit, can you check again? my linter edited those and i've reverted them to the original, it should be the same as main now